concent_api/core/tests/test_unit_transfer_operation.py

```diff
@@ -47,31 +53,74 @@ def test_properly_work_of_request_upload_status_function(self):
                 subtask_id                      = '1',
             )
         )
-        file_transfer_token = create_file_transfer_token(
-            report_computed_task,
-            self.REQUESTOR_PUBLIC_KEY,
-            FileTransferToken.Operation.upload,
-        )
 
         with mock.patch('core.transfer_operations.send_request_to_cluster_storage', mock_send_request_to_cluster_correct_response):
-            cluster_response = request_upload_status(
-                file_transfer_token,
-                report_computed_task,
-            )
+            cluster_response = request_upload_status(report_computed_task)
 
         self.assertEqual(cluster_response, True)
 
         with mock.patch('core.transfer_operations.send_request_to_cluster_storage', mock_send_incorrect_request_to_cluster_incorrect_response):
-            cluster_response_2 = request_upload_status(
-                file_transfer_token,
-                report_computed_task,
-            )
+            cluster_response_2 = request_upload_status(report_computed_task)
 
         self.assertEqual(cluster_response_2, False)
 
         with self.assertRaises(UnexpectedResponse):
             with mock.patch('core.transfer_operations.send_request_to_cluster_storage', mock_send_incorrect_request_to_cluster_unexpected_response):
-                request_upload_status(
-                    file_transfer_token,
-                    report_computed_task,
+                request_upload_status(report_computed_task)
+
+
+@override_settings(
+    CONCENT_PRIVATE_KEY=CONCENT_PRIVATE_KEY,
+    CONCENT_PUBLIC_KEY=CONCENT_PUBLIC_KEY,
+)
+class FileTransferTokenCreationTest(TestCase):
+    def setUp(self):
+        self.time = datetime.datetime.strptime("2017-11-17 10:00:00", "%Y-%m-%d %H:%M:%S")
+        self.deadline = 10
+        self.authorized_client_public_key = b'7' * 64
```

Comment:
Should it be bytes or string ?

Reply:
`bytes`. `authorized_client_public_key` is not base64-encoded. Just like the public keys in `TaskToCompute`.